templates/ai/data-science.html

```diff
@@ -27,8 +27,8 @@ <h1>Data Science Stack on Ubuntu</h1>
         </div>
         <hr />
         <a class="p-button--positive"
-           href="https://documentation.ubuntu.com/data-science-stack/en/latest/tutorial/">Try the Beta</a>
-        <a href="/engage/data-science-tools">Learn more about data science tools on Ubuntu&nbsp;&rsaquo;</a>
+           href="https://documentation.ubuntu.com/data-science-stack/en/latest/tutorial/">Try it out</a>
+        <a href="https://ubuntu.com/engage/data-science-stack">Watch the webinar to learn more&nbsp;&rsaquo;</a>
```

Comment:
I think this can also stay as a relative path :)
```suggestion
        <a href="/engage/data-science-stack">Watch the webinar to learn more&nbsp;&rsaquo;</a>
```